woocommerce/wc-admin
# Home Screen: Add Store Management (Quick Links) card

Fixes woocommerce/woocommerce-admin#4082

This PR adds the new "Store Management" (Quick Links) card to the home screen.

### Accessibility




### Screenshots

<img width="1263" alt="ScreenCapture at Sat May 16 06:44:50 EDT 2020" src="https://user-images.githubusercontent.com/2098816/82117964-ecd0d980-9741-11ea-92e7-97220e69a79d.png">

### Detailed test instructions

- Enable debugging of Tracks in your browser console:

```
localStorage.setItem( 'debug', 'wc-admin:*' );
```

- View the new home screen.
- Verify that the "Store Management" card is shown.
- Click items and verify that they navigate to the correct locations (from woocommerce/woocommerce-admin#4082) 
    - When the user clicks "Market my store" he will be redirected to WooCommerce - Marketing
    - When the user clicks "Add products" he will be redirected to WooCommerce - Products
    - When the user clicks "Personalize my store" he will be redirected to Appearance - Customize
    - When the user clicks "Shipping settings" he will be redirected to WooCommerce - Settings - Shipping
    - When the user clicks "Tax settings" he will be redirected to WooCommerce - Settings - Tax
    - When the user clicks "Payment settings" he will be redirected to WooCommerce - Settings - Payments
    - When the user clicks "Edit store details" he will be redirected to WooCommerce - Settings - General
    - When the user clicks "View my store" he will be redirected to the site frontend
- Verify that `wcadmin_home_quick_links_click` Tracks events are recorded, with the `task_name` custom prop set appropriately, every time an item in the list is clicked.

### Unit tests

```
npm test -- -i client/quick-links/test/index.js
```

### Known issues

- Icons do not match design. (/cc @jameskoster Where can I find the correct icons to use?)
- Card always shows on home screen. It should only appear when the "Set up your store and start selling" task list card has been completed or manually dismissed. This will be addressed after the task list card has been merged (#4378)
- Card has incorrect ARIA roles (#4387). This should be addressed separately, as it is a problem with the underlying `<List>` component.
- Support link always goes to premium support. This should be addressed separately, as it is a core problem with support links in general (https://github.com/woocommerce/woocommerce/issues/25442), and a solution should be implemented to make sure that support links are consistent throughout the product.
- Storybook does not show styles correctly for `<QuickLinks>` component



### Changelog Note:



Enhancement: New "Store Management" quick links card on WooCommerce home screen.
